src/pandablocks/responses.py

```diff
@@ -196,6 +196,8 @@ class EndReason(Enum):
 
     #: Experiment completed by falling edge of ``PCAP.ENABLE```
     OK = "Ok"
+    #: Experiment manually completed by ``DATA:CAPTURE``
+    MANUALLY_STOPPED = "Manually stopped"
```

Comment:
Can you put these at the bottom, after `START_DATA_MISMATCH` as they are not generated by the server